src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+
+    local batch_input
+    batch_input=$(find inputs/basecounts/ -type f -name '*_basecount.hdf5' -exec basename {} \; \
+        | sort | sed 's/^/--input \/data\/basecounts\//g')
+
+    SECONDS=0
+    docker run \
+        -v /home/dnanexus/inputs:/data \
+        "$GATK_image" gatk DetermineGermlineContigPloidy \
+        -L /data/beds/filtered.interval_list \
+        -imr OVERLAPPING_ONLY \
+        $DetermineGermlineContigPloidy_args \
```

Shellcheck

[warning] DetermineGermlineContigPloidy_args is referenced but not assigned.

(SC2154)